Tchap/Modules/PublicRooms/PublicRoomsDataSource.swift

```diff
@@ -146,11 +146,11 @@ extension PublicRoomsDataSource: UITableViewDataSource {
             
             let room = rooms[indexPath.row]
             
-            if let publicRoomCell = tableView.dequeueReusableCell(withIdentifier: PublicRoomTableViewCell.defaultReuseIdentifier(), for: indexPath) as? PublicRoomTableViewCell {
-                publicRoomCell.render(room, withMatrixSession: self.session)
+            if let publicRoomCell = tableView.dequeueReusableCell(withIdentifier: PublicRoomsCell.defaultReuseIdentifier(), for: indexPath) as? PublicRoomsCell {
```

Comment:
you could use `let publicRoomCell = tableView .dequeueReusableCell(for indexPath: IndexPath, cellType: PublicRoomsCell.self)` if PublicRoomsCell conform to NibReusable